cocos/audio/apple/AudioPlayer.mm

```diff
@@ -305,15 +330,15 @@ of this software and associated documentation files (the "Software"), to deal
 
             if (!_needWakeupRotateThread)
             {
-                _sleepCondition.wait_for(lk,std::chrono::milliseconds(75));
+                _sleepCondition.wait_for(lk,std::chrono::milliseconds(rotateSleepTime));
```

Comment:
the sleep time in rotate buffer thread shouldn't been hard coded, it depends on `QUEUEBUFFER_TIME_STEP`